.github/workflows/unit-tests.yml

```diff
@@ -20,3 +20,57 @@ jobs:
         uses: actions/checkout@v4
       - name: Run unit tests
         run: go test -race -short
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.23.x
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: vet .
+        run: go vet ./...
+
+      - name: vet ./examples
+        working-directory: ./examples
+        run: go vet ./...
+
+      - name: vet ./benchmarks
```

Comment:
I removed all benchmarks related changes.

Reply:
I'll send the benchmarks related PR once this PR gets merged, because github doesn't do stacked PR-s well. For the time being those changes are just here https://github.com/egonelbre/go-sql-spanner/pull/2.